js/src/duplicate-post-strings.js

```diff
@@ -1,20 +1,43 @@
 /* global duplicatePostStrings */
 
 import { createInterpolateElement } from "@wordpress/element";
+import { Button } from "@wordpress/components";
 import { __, setLocaleData } from "@wordpress/i18n";
+import { dispatch, subscribe } from "@wordpress/data";
+import { redirectOnSaveCompletion } from "./duplicate-post-functions";
+
+const saveAndCompare = () => {
+	dispatch( 'core/editor' ).savePost();
```

Comment:
Since we have now a `js/src/duplicate-post-functions.js` file, maybe it would be cleaner to move this function there and then import it here. This "strings" file should contain as less logic as possible. Let's consider this for future refinements.